test/ui/cypress/code/knative/serving/showcase.js

```diff
@@ -26,16 +26,22 @@ class ShowcaseKservice {
 
   makeRequest(baseUrl) {
     const req = {
-      method: 'OPTIONS',
+      method: 'GET',
       url: baseUrl,
       retryOnStatusCodeFailure: true,
-      failOnStatusCode: true
+      failOnStatusCode: true,
+      headers: {'user-agent': `Cypress/${Cypress.version}`}
     }
     cy.request(req).then((response) => {
       expect(response.status).to.eq(200)
-      expect(response.body).to.have.property('version')
-      expect(response.body.group).to.include('openshift')
-      expect(response.body.artifact).to.eq('knative-showcase')
+      expect(response.headers)
+        .to.have.property('content-type')
+        .that.matches(/^application\/json(?:;.+)?$/)
+      expect(response.headers).to.have.property('x-version')
+      expect(response.headers).to.have.property('x-config')
```

Comment:
I suppose the headers are not case sensitive because the other PR was sending `X-Config` back: https://github.com/openshift-knative/showcase/pull/10/files#diff-b5a606a3156da4a52ad2e4b50f8ef57e2752ee7cf9abe18e333a8a347b906ad8R70

Reply:
They are actually lower cased here. Even if backend send them in upper case.

Such change was done in web standards couple of years ago, and more and more software is using lower cased headers.

See: https://stackoverflow.com/a/41169947